docs/integrating-into-custom-elements.md

```diff
@@ -0,0 +1,61 @@
+# Integrating MDLv2 into custom elements
```

Comment:
Possibly best to keep it as a section in the frameworks guide, then.

I guess the thing that confuses me is that, unlike frameworks, where you can have an implementation of each component on top of each framework, there would only be a single custom elements implementation for each component. That's why a guide explaining how to do it feels odd; once it's done and published, it's done; it's not something folks would need to keep doing and thus need documentation for.